src/travix/Travix.hx

```diff
@@ -619,6 +668,9 @@ class Travix {
       case 'php': t.doPhp();
       case 'python': t.doPython();
       case 'cs': t.doCs();
+      case 'lua': 
+        if(haxeVersion < "3.3.0") die('lua requires haxe >= 3.3.0');
```

Comment:
Maybe so, my rationale is that if I'm trying to run lua tests and it's not supported, I'd like to know what's going on.